cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/hvdc/AcDcConverterConversion.java

```diff
@@ -71,16 +79,21 @@ public void convert() {
             // TODO: There are two modes of control: dcVoltage and activePower
             // For dcVoltage, setpoint is targetUdc,
             // For activePower, setpoint is targetPpcc
+
             LccConverterStationAdder adder = voltageLevel().newLccConverterStation()
-                    .setLossFactor(0.0f)
-                    .setPowerFactor(0.8f);
+                .setLossFactor((float) this.lossFactor)
```

Comment:
Hi José Antonio, I have performed some tests on small grid case (with HVDC lines) and it seems that the attributes of the LCC converters are nor correctly set. I understand that the power factor is at its default value. It seems that the loss factor is equal too zero in IIDM in my tests, but the ````poleLoss```` in CGMES is not equal to zero. Can you check again ?

Reply:
It may be linked to the fact that `lossFactor` is only set when both are not `NaN` in CGMES but I think it happens that only one side is set (cf. my question above). I think we should retrieve as much information as possible and only set to default values when there is no info.

Reply:
Solved. Loss Factor is always calculated. If lossFactor is NaN it is fixed to 0.0